packages/init/test/SliceMachineInitProcess-run.test.ts

```diff
@@ -183,6 +183,7 @@ it("runs it all", async (ctx) => {
 	const initProcess = createSliceMachineInitProcess({
```

Comment:
If you can add a test when slicemachine should be lanch it's perfect

Reply:
Done by injecting "Y" in prompts _but_ I can't see the stdout from the start-slicemachine command